src/data/analyses.js

```diff
@@ -0,0 +1,121 @@
+var MAP = {
```

Comment:
Right, the only thing I can think of it's a npm module, but you are right, it's not the time.